src/runtime/GlobalObjectBuiltinArray.cpp

```diff
@@ -1835,7 +1845,8 @@ void GlobalObject::installArray(ExecutionState& state)
                                                                ObjectPropertyDescriptor(new FunctionObject(state, NativeFunctionInfo(state.context()->staticStrings().next, builtinArrayIteratorNext, 0, nullptr, NativeFunctionInfo::Strict)), (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::WritablePresent | ObjectPropertyDescriptor::ConfigurablePresent)));
     m_arrayIteratorPrototype->defineOwnPropertyThrowsException(state, ObjectPropertyName(state, Value(state.context()->vmInstance()->globalSymbols().toStringTag)),
                                                                ObjectPropertyDescriptor(Value(String::fromASCII("Array Iterator")), (ObjectPropertyDescriptor::PresentAttribute)(ObjectPropertyDescriptor::ConfigurablePresent)));
-
+    m_arrayPrototype->defineOwnPropertyThrowsException(state, ObjectPropertyName(state.context()->staticStrings().unscopables),
+                                                       ObjectPropertyDescriptor(blackList));
```

Comment:
`This property has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: true }.`

Reply:
I think this issue is still present.